EddieHubCommunity/support
# fix: Unnecessary Spaces And Grammar

closes #1582 

#### What does this PR do?

resolves #1582 

#### Description of Task to be completed?

fix #1582 

#### How can this be manually tested?

See the diff:

```diff
diff --git a/youtube/generic.md b/youtube/generic.md
index 71ecb86..ee69d62 100644
--- a/youtube/generic.md
+++ b/youtube/generic.md
@@ -1,49 +1,57 @@
-...
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
 
+MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!
 
-## LEARN MORE
+## YouTube Shorts
+I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1
 
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
+## Newsletter
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls
 
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
+ 
+\1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+\2. Choose The Issue Type Of Invite Me To The Organization
+\3. Choose The Issue Type Of Invite Me To The Organization
+\4. Open The Issue
+\5. You get an invitation to the community by our GitHub Action 🎉
 
-## SERVICES I USE AND RECOMMEND
+---
 
-Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
-
-- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
-- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
-- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
-- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
-
-## COMMUNITY SPONSORS
+## Community Sponsors
 
-A BIG thank you to my GitHub Sponsors:
-***
 Stephen Mount http://github.com/stemount
 Nicholas Carrigan https://github.com/nhcarrigan
-***
+Santosh Yadav https://github.com/santoshyadavdev
+Andrew Cunliffe https://github.com/andrew-cunliffe
+Alexander Stoichkov https://github.com/SashoStoichkov
+Allan Regush https://github.com/AllanRegush
 
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
+---
 
-## FIND ME ON OTHER SOCIAL PLATFORMS
+## Service I Use and Recommend
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
 
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+Live streaming with Streamyard: http://links.eddiejaoude.io/t/pYlzGjcta1a4
+Cloud hosting with Digital Ocean, you will get $100 FREE credit: http://links.eddiejaoude.io/t/EAaCRWhsnZlF
+NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
+Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
+
+---
+ 
+## Find me on social media
+
+Website: https://www.eddiejaoude.io 
+GitHub: https://github.com/eddiejaoude
+YouTube: https://youtube.com/eddiejaoude
+Twitter: https://twitter.com/eddiejaoude
+Instagram: https://instagram.com/eddiejaoude
+LinkedIn: https://linkedin.com/in/eddiejaoude
+GitHub Community Org: https://github.com/EddieHubCommunity
+
+## Further
+Join Our Conversation At Discord: http://discord.eddiehub.org/
```

#### Any background context you want to provide?

Duplicate of #1615 made due to some issue see in the pull request comments

#### What is the relevant issue link?

#1582

#### Screenshots (if appropriate)

<img src="https://user-images.githubusercontent.com/30869493/117687590-3bc81800-b1b8-11eb-8b94-6b539aef0d42.jpg" width="300">
